docs/source/using/troubleshooting.adoc

```diff
@@ -243,3 +243,71 @@ Workaround: Avoid installing packages or storing large files in the root filesys
 Instead, you can create a sub-directory in the *_mnt/sda1/_* persistent storage volume or define and mount xref:../using/host-folders.adoc#[host folders] that can share storage space between the host and the Minishift VM.
 
 If you want to perform development tasks inside the Minishift VM, it is recommended that you use containers, which are stored in persistent storage volumes, and reuse the xref:../using/docker-daemon.adoc#[Minishift Docker daemon].
+
+
+[[minshift-startup-check-failed]]
+== Minishift startup check failed
+
+While Minishift starts, it runs several startup checks to make sure that the Minishift VM and the OpenShift Cluster are able to start without any issues.
+If any configuration is incorrect or missing, the startup checks fail and Minishift does not start.
+
+The following sections describe the different startup checks.
+
+Driver plug-in configuration::
+One of the startup checks verifies that the relevant driver plug-in is configured correctly.
+If this startup check fails, review the xref:../getting-started/setting-up-driver-plugin.adoc#[setting up the driver plug-in] topic and configure the driver.
+
```

Comment:
Need to add a + sign here to bing the following paragraph to the sub-section (if you click the View button in the diff tab of the PR you'll see that the alignment is off).